iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/RoutePreview/RoutePreviewStatus/BaseRoutePreviewStatus.swift

```diff
@@ -84,7 +84,19 @@ final class BaseRoutePreviewStatus: SolidTouchView {
     }
   }
 
+  @objc static func setImagePadding(_ padding: CGFloat, for button: UIButton) {
```

Comment:
Such kind of static methods from the some class should not be used for modifying the instances of another classes.

An extension for the UIButton should be created:
`UIButton+SetImagePadding.swift`